src/range_proof.rs

```diff
@@ -751,54 +750,26 @@ where
 
     // Helper function to decompress Li
     fn li_decompressed(&self) -> Result<Vec<P>, ProofError> {
-        if self.li.is_empty() {
-            Err(ProofError::InvalidArgument("Vector 'L' not assigned yet".to_string()))
```

Comment:
I would not remove this safety check

Reply:
It definitely won't be needed once #57 is merged and the inner-product prover is baked into the range prover.

Although to be honest, I don't think it's needed with the current setup either. Verification won't succeed on an empty vector anyway, and returning empty vectors seems like the correct generic behavior of the getters.